src/poprox_storage/concepts/manifest.py

```diff
@@ -85,34 +94,43 @@ def manifest_to_experiment(manifest: ManifestFile) -> Experiment:
     start_date = manifest.experiment.start_date or (date.today() + timedelta(days=1))  # noqa: DTZ011
     end_date = start_date + convert_duration(manifest.experiment.duration)
 
+    owner = Team(
+        team_id=manifest.owner.team_id,
+        team_name=manifest.owner.team_name,
+        members=manifest.owner.members,
```

Comment:
Is the implication here that the manifest will contain a list of member UUIDs? Do we like this, or should we have the manifest list emails and then look up the UUIDs as part of ingress here?

Reply:
Also -- from a type-safety standpoint (which might be uninteresting FWIW) would this be `list[str]` or `list[UUID]` at this point? I feel like it would be `list[str]` but the `Team` class has `list[UUID]`?

Not sure we care about that, but it occurred to me that it's worth checking.

Reply:
> Is the implication here that the manifest will contain a list of member UUIDs?
Yes!

> Do we like this, or should we have the manifest list emails and then look up the UUIDs as part of ingress here?
No! I do not like this but it allowed me to punt on integrating account lookups, which complicate the picture as far as testing goes.

> would this be list[str] or list[UUID] at this point? I feel like it would be list[str] but the Team class has list[UUID]?
It should be `list[UUID]` but it's actually `list[str]`. I'll fix this!

Reply:
> The conversion from string to UUID happens when Pydantic turns the manifest JSON into model objects.

Neat!